youtube-gpt-client/src/pages/video-summary/[videoId].tsx

```diff
@@ -12,24 +12,60 @@ import {
   Heading,
   Text,
   Button,
+  Spinner,
 } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
 
 interface Props {
   summaryData: SummaryData;
 }
 
-const VideoSummaryDetail = ({ summaryData }: Props) => {
+const VideoSummaryDetail = ({ _nextI18Next: { initialLocale } }: any) => {
```

Comment:
You don't need to pass this as parameter, you can use the i18n object to get the language something like this:
```const { t, i18n } = useTranslation();``` and retrieve using this: ```i18n?.language```

Reply:
ty